js/src/util/storage.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * A safe localStorage wrapper
+ */
+class Storage {
```

Comment:
Instead of doing this, perfer the use of store, i.e. `import store from 'store';`

1. Consistent with what we use elsewhere instead of adding yet another module
2. No need for try/catch
3. It does nice things like auto to/from JSON